fetch.bs

```diff
@@ -6939,10 +6942,19 @@ constructor steps are:
 
    <li><p>Unset <var>request</var>'s <a for=request>history-navigation flag</a>.
 
+   <li><p>Set <var>request</var>'s <a for=request>origin</a> to
+   "<code>client</code>"
+
    <li><p>Set <var>request</var>'s <a for=request>referrer</a> to
    "<code>client</code>"
 
    <li><p>Set <var>request</var>'s <a for=request>referrer policy</a> to the empty string.
+
+   <li><p>Set <var>request</var>'s <a for=request>URL list</a> to an empty
+   <a for=/>list</a>.
+
+   <li><p><a for=list>Append</a> <var>request</var>'s <a for=request>URL</a> to
+   <var>request</var>'s <a for=request>URL list</a>.
```

Comment:
I'm not sure how this works. Request's URL is a pointer to the first item of request's URL list. It seems you would have to store it in a variable before you reset request's URL list, right? But also, you probably want the last item in request's URL list? I.e., request's current URL.

Reply:
As I linked above, its not defined that way.  The first entry in the URL list contains a copy of the request URL.

You are right, though, I need to use the current URL instead.

Reply:
I handled this by storing the current URL in the request's URL and then appending to the new list.